.github/compatibility-test-matrices/main/client-chain-a.json

```diff
@@ -32,4 +31,4 @@
   "chain-image": [
     "ghcr.io/cosmos/ibc-go-simd"
   ]
-}
+}
```

Comment:
ideally this noise should disappear in the future, the script currently trims the last empty line!